ops/ansible/roles/bfd-pipeline/tasks/etl.yml

```diff
@@ -1,16 +1,5 @@
 ---
 
-- name: Install Pre-requisites
-  yum:
-    name: "{{ item }}"
-    state: present
-  with_items:
-    # Needed to run the ETL service.
-    - java-1.8.0-openjdk
-  become: true
-  tags: 
-    - pre-ami
-
```

Comment:
Java is already installed during the platinum build, not required here.

Reply:
FYI: Undoing this as part of the capsule changes. Two reasons:

1. We have tests that also use the role.
2. It's not listed as a pre-req for the role, so the role should provide it.